aten/src/ATen/cpu/vec/vec256/vec256_int.h

```diff
@@ -74,7 +74,7 @@ class Vectorized<int64_t> : public Vectorizedi {
     return _mm256_blendv_epi8(a.values, b.values, mask.values);
   }
   template <typename step_t>
-  static Vectorized<int64_t> arange(int64_t base = 0, step_t step = static_cast<step_t>(1)) {
+  static Vectorized<int64_t> arange(double base = 0, step_t step = static_cast<step_t>(1)) {
```

Comment:
Doesn't double have smaller range than full int64? (53 bits instead of 64 bits, right?) (especially for very large values of int64?)

Reply:
Yes. You're correct. We will not modify the base type here, since vectorized implementation has been removed.